disperser/dataapi/prometheus_client.go

```diff
@@ -12,6 +12,9 @@ import (
 const (
 	// maxNumOfDataPoints is the maximum number of data points that can be queried from Prometheus based on latency that this API can provide
 	maxNumOfDataPoints = 3500
+
+	// Calculate the average over this number of minutes for signing rate
+	siningRateRangeVectorMinutes = 9
```

Comment:
Why the number 9?

Reply:
it's just copied the existing throughput time series query, but I'm not sure what was the calculation behind it:) it looks arbitrary to me

for signing rate, we may probably want to be a bit more responsive, so we could try 5m